recipes/arm-variant/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: arm-variant
+  version: 1.0.0
+
+build:
+  number: 0
+  string: "{{ arm_variant_type }}"
+  skip: true  # [not aarch64]
+
+test:
+  commands:
+    - exit 0
```

Comment:
Is there a reason to have an empty test like this?

Reply:
To keep the linter happy :)